test/start-kubernetes.sh

```diff
@@ -0,0 +1,301 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+
+TEST_DIRECTORY=${TEST_DIRECTORY:-$(dirname $(readlink -f $0))}
+TEST_CONFIG=${TEST_CONFIG:-${TEST_DIRECTORY}/test-config.sh}
+DEPLOYMENT_SUFFIX=${DEPLOYMENT_SUFFIX:-govm}
+CLOUD_USER=${CLOUD_USER:-clear}
+CLUSTER=${CLUSTER:-${CLOUD_USER}-${DEPLOYMENT_SUFFIX}}
+DEPLOYMENT_ID=${DEPLOYMENT_ID:-pmem-csi-${CLUSTER}}
+GOVM_YAML=${GOVM_YAML:-$(mktemp --suffix $DEPLOYMENT_ID.yml)}
+REPO_DIRECTORY=${REPO_DIRECTORY:-$(dirname $TEST_DIRECTORY)}
+TEST_DIRECTORY=${TEST_DIRECTORY:-$(dirname $(readlink -f $0))}
+RESOURCES_DIRECTORY=${RESOURCES_DIRECTORY:-${REPO_DIRECTORY}/_work/resources}
+WORKING_DIRECTORY="${WORKING_DIRECTORY:-${REPO_DIRECTORY}/_work/${CLUSTER}}"
+source ${TEST_CONFIG}
+NODES=( $DEPLOYMENT_ID-master
+	$DEPLOYMENT_ID-worker1
+	$DEPLOYMENT_ID-worker2
+	$DEPLOYMENT_ID-worker3)
+CLOUD="${CLOUD:-true}"
+FLAVOR="${FLAVOR:-medium}"
+SSH_KEY="${SSH_KEY:-${RESOURCES_DIRECTORY}/id_rsa}"
+SSH_PUBLIC_KEY="${SSH_KEY}.pub"
+IMAGE_TAG="${IMAGE_TAG:-canary}"
+EFI="${EFI:-true}"
+KVM_CPU_OPTS="${KVM_CPU_OPTS:-\
+		-m 2G,slots=${TEST_MEM_SLOTS:-2},maxmem=34G -smp 4 \
+		-machine pc,accel=kvm,nvdimm=on}"
+EXTRA_QEMU_OPTS="${EXTRA_QWEMU_OPTS:-\
+		-object memory-backend-file,id=mem1,share=${TEST_PMEM_SHARE:-on},\
+		mem-path=/data/nvdimm0,size=${TEST_PMEM_MEM_SIZE:-32768}M \
+		-device nvdimm,id=nvdimm1,memdev=mem1,label-size=${TEST_PMEM_LABEL_SIZE:-2097152} \
+		-machine pc,nvdimm}"
+CLOUD_USER=${CLOUD_USER:-clear}
+CLOUD_IMAGE="${CLOUD_IMAGE:-$(\
+		curl -s https://download.clearlinux.org/image/latest-images |\
+		awk '/cloud.img/ {gsub(".xz",""); print $0}')}"
```

Comment:
here we take CLOUD_IMAGE name, strip off .gz, and then numerous time add it back when using this variable, but only once we use the CLOUD_IMAGE as it is (down when creating govm description). It would reduce changes if we maintained the name with .xz part and and remove it in that one occasion

Reply:
I changed the code to maintain the .xz